Makefile

```diff
@@ -57,6 +57,7 @@ endif
 .PHONY: server
 server:
 ifneq ($(HAS_SERVER),)
+	$(info Are you sure you want to build the plugin with an id of "com.mattermost.plugin-starter-template"? Consider editing plugin.json to configure your project.)
```

Comment:
Wondering if using `$(info message)` is the right approach or should it be changed to `$(warning message)`?

Reply:
Only my system using `warning` outputs the line number, which is not so helpful.

Maybe adding a newline after the output helps to separate it from the following log lines.